lib/ntlm/ntlm.c

```diff
@@ -718,6 +718,9 @@ heim_ntlm_decode_type1(const struct ntlm_buf *buf, struct ntlm_type1 *data)
 /**
  * Encodes an ntlm_type1 message.
  *
+ *     [MS-NLMP] 2.2.1.1 NEGOTIATE_MESSAGE
+ *     https://docs.microsoft.com/en-us/openspecs/windows_protocols/ms-nlmp/
+ *
```

Comment:
I cannot reasonably review this change nor do I have a way of testing it.

Reply:
The linked spec does say this (`SHOULD`).